### PR TITLE
OZ-437: Added ability to run Ozone KH tests on a preferred environment via GitHub Actions

### PR DESCRIPTION
**Pull request by kdaud**
Ticket → [OZ-437](https://mekomsolutions.atlassian.net/browse/OZ-437)

[OZ-437]: https://mekomsolutions.atlassian.net/browse/OZ-437?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,9 +65,10 @@ e2e
...
-## Guide for writing the tests
+## Guide for writing tests
 
-When writing a new test case, you need to create a new spec in `./e2e/tests`
+When writing a new test case, create a spec in `./e2e/tests`
 
-## Github Action integration
+## GitHub Actions integration
 The e2e.yml workflow is made up of one job that is triggered by PRs, and on a push.
+The env.yml workflow is made up of one job that supports selection of a preferred environment at runtime.
```
Comment (Ruhanga) [inline README.md:74]: Lets adopt the docs here following [this comment template](https://github.com/ozone-his/ozone-e2e-pro/pull/60#pullrequestreview-1727373150).

**Review by Ruhanga — COMMENTED** [review]
`env.yml` doesn't seem to make sense based on what it is actually accomplishing. Could we rename it to something like `runE2ETestsOnProvidedEnv.yml`

File: README.md (hunk 1 above)
Comment (kdaud) [reply README.md]: → [6fe1183](https://github.com/mekomsolutions/ozone-e2e-kh/pull/24/commits/6fe11838ef9ca0dac2b955993dc475fe48ebb593)

**Review by Ruhanga — APPROVED** [review]
LGTM!

**Pull request merged**